addons/analytic/models/analytic_account.py

```diff
@@ -10,7 +10,6 @@ class AccountAnalyticAccount(models.Model):
     _name = 'account.analytic.account'
     _inherit = ['mail.thread']
     _description = 'Analytic Account'
-    _order = 'code, name asc'
```

Comment:
The message about this is quite unclear.
Why are you not ordering by default, but only on `view_account_analytic_account_list`?